sqlite/src/parse.y

```diff
@@ -1329,6 +1329,10 @@ term(A) ::= CTIME_KW(OP). {
   A = sqlite3ExprFunction(pParse, 0, &OP, 0);
 }
 
+term(A) ::= GUID_FUNC(OP). {
```

Comment:
Could you please add the special comdb2 %ifdef / %endif pair here to help the automated tooling?

Reply:
Done. Thanks.